pydis_site/apps/api/models/bot/whitelist.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from pydis_site.apps.api.models.utils import ModelReprMixin
+
+
+class Whitelist(ModelReprMixin, models.Model):
+    """Whitelisted item for a type."""
+
+    TYPE_CHOICES = (
+        ("invite", "invite"),
+        ("extension", "extension"),
+        ("channel", "channel"),
+        ("role", "role")
```

Comment:
Capitalise the second element of each tuple - those are the human-readable names.
```suggestion
        ("invite", "Invite"),
        ("extension", "Extension"),
        ("channel", "Channel"),
        ("role", "Role")
```
Not sure if this requires a migration.